packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2142,6 +2230,17 @@ module.exports = function (log, config, oauthdb) {
     const translatorParams = { productName };
     const subject = translator.gettext('%(productName)s payment received');
 
+    let paymentProrated;
+    let showProratedAmount = false;
+    if (typeof paymentProratedInCents !== 'undefined') {
```

Comment:
Could this not simply be `if (paymentProratedInCents != null) {`?

Reply:
If I recall (and I'll look to double check), the `paymentProratedInCents` doesn't get included at all if there's no proration. So it's undefined altogether. We had a `!!` in there, but linting was complaining about it

Reply:
Fair enough - I figured there must've been a reason to do this. Carry on!